@readalongs/ngx-web-component doesn't actually work outside this repo

Trying to recreate the angular-demo standalone:
```
npm install @angular/cli@15
npx ng new test-ngx-web-component
cd test-ngx-web-component
npm install @readalongs/ngx-web-component
```
Oops! That doesn't actualy work, because npm dependency hell:
```
(a bunch of absolute horror npm output snipped)
npm ERR! While resolving: test-ngx-web-component@0.0.0
npm ERR! Found: rxjs@7.8.1
(more horror)
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"~7.5.0" from @readalongs/ngx-web-component@1.2.1
(second verse, same as the first, after fixing the rxjs dependency)
npm ERR! While resolving: test-ngx-web-component@0.0.0
npm ERR! Found: zone.js@0.12.0
(etc)
npm ERR! Could not resolve dependency:
npm ERR! peer zone.js@"^0.11.8" from @readalongs/ngx-web-component@1.2.1
```
Well, let's frob package.json in our test app as we have learned to do (probably this can be fixed in web-component...):
```diff
-    "rxjs": "~7.8.0",
+    "rxjs": "~7.5.0",
     "tslib": "^2.3.0",
-    "zone.js": "~0.12.0"
+    "zone.js": "~0.11.8"
```
Okay, now at least we can `npm install`.  So let's copy over the stuff from the angular demo app and see if we can make it work.  First we add the stuff necessary to `src/app/app.module.ts`:
```diff
 import { AppComponent } from './app.component';
+import { NgxRAWebComponentModule } from "@readalongs/ngx-web-component";
+import { defineCustomElements } from "@readalongs/web-component/loader";
+
+defineCustomElements();
 
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,
+    NgxRAWebComponentModule,
```
Does it compile? NO!
```
npx ng serve
(...snip...)
./src/app/app.module.ts:4:0-72 - Error: Module not found: Error: Can't resolve '@readalongs/web-component/loader' in '/home/dhd/work/ReadAlongs/testng15/test-ngx-web-component/src/app'
./node_modules/@readalongs/ngx-web-component/esm2020/lib/ngx-web-component.module.mjs:4:0-72 - Error: Module not found: Error: Can't resolve '@readalongs/web-component/loader' in '/home/dhd/work/ReadAlongs/testng15/test-ngx-web-component/node_modules/@readalongs/ngx-web-component/esm2020/lib'
Error: src/app/app.module.ts:6:38 - error TS2307: Cannot find module '@readalongs/web-component/loader' or its corresponding type declarations.
6 import { defineCustomElements } from "@readalongs/web-component/loader";
```
Strange, since the `@readalongs/web-component` is correctly defined in the `peerDepedencies` and, indeed, `npm` has installed it as expected...

This seems to have something to do with the convoluted way in which the declarations files are structured.  First of all the main types declaration file in `package.json` is `dist/types/components.d.ts` which is almost certainly wrong since there's no possible way you can get to `defineCustomElements` from there.

For this or whatever other reason TypeScript isn't able to resolve "@readalongs/web-component/loader" to `node_modules/@readalongs/web-component/dist/loader/index.d.ts`.  Compounding this is, maybe, it is trying to either import `node_modules/@readalongs/web-component/dist/cjs/loader.cjs.js` or maybe `node_modules/@readalongs/web-component/dist/esm/loader.js` 

The level of complexity in how TypeScript and NPM actually resolve imports is too much for my simple snake brain to handle, so unfortunately I have no proposed solution yet...

--- a/packages/angular-demo/src/assets/ej-fra.readalong
+++ b/packages/angular-demo/src/assets/ej-fra.readalong
@@ -6,7 +6,7 @@
     <text xml:lang="fra" id="t0">
         <body id="t0b0">
             <div type="page" id="t0b0d0">
-            <graphic url="http://localhost:5000/ej-fra/avatar.png" id="t0b0d0graphic0"/>
+            <graphic url="avatar.png" id="t0b0d0graphic0"/>
                 <p id="t0b0d0p0">
                     <s id="t0b0d0p0s0"><w id="t0b0d0p0s0w0" time="0.455" dur="1.165">Bonjour</w>.</s>
                     <s id="t0b0d0p0s1"><w id="t0b0d0p0s1w0" time="1.620" dur="0.070">Je</w> <w id="t0b0d0p0s1w1" time="1.690" dur="0.070">m</w>'<w id="t0b0d0p0s1w2" time="1.760" dur="0.240">appelle</w> <w id="t0b0d0p0s1w3" time="2.000" dur="0.220">Éric</w> <w id="t0b0d0p0s1w4" time="2.220" dur="0.370">Joanis</w>.</s>

--- a/packages/ngx-web-component/package.json
+++ b/packages/ngx-web-component/package.json
@@ -11,9 +11,9 @@
     "@angular/platform-browser": "^15.2.4",
     "@angular/platform-browser-dynamic": "^15.2.4",
     "@angular/router": "^15.2.4",
-    "rxjs": "~7.5.0",
+    "rxjs": "~7.8.0",
     "tslib": "^2.3.0",
-    "zone.js": "^0.11.8"
+    "zone.js": "~0.12.0"
   },
   "private": false,
   "nx": {

--- a/packages/studio-web/package.json
+++ b/packages/studio-web/package.json
@@ -50,13 +50,13 @@
     "jszip": "^3.10.1",
     "mime": "^4.0.1",
     "ngx-toastr": "^16.1.0",
-    "rxjs": "~7.5.0",
+    "rxjs": "~7.8.0",
     "shepherd.js": "^11.0.1",
     "soundswallower": "^0.6.3",
     "standardized-audio-context": "^25.3.41",
     "tslib": "^2.3.0",
     "unidecode": "^1.0.1",
-    "zone.js": "^0.11.8"
+    "zone.js": "~0.12.0"
   },
   "singleFileBundleVersion": "1.2.1",
   "singleFileBundleTimestamp": "2024-04-02+15-07-59"

--- a/packages/web-component/package.json
+++ b/packages/web-component/package.json
@@ -12,7 +12,8 @@
   "unpkg": "dist/bundle.js",
   "private": false,
   "files": [
-    "dist/"
+    "dist/",
+    "loader/"
   ],
   "scripts": {
     "build": "stencil build --docs",

--- a/packages/web-component/package.json
+++ b/packages/web-component/package.json
@@ -30,7 +30,7 @@
   "dependencies": {
     "audio-recorder-polyfill": "^0.4.1",
     "howler": "^2.2.3",
-    "rxjs": "^6.6.7"
+    "rxjs": ">=6.6.7"
   },
   "devDependencies": {
     "@stencil/angular-output-target": "^0.4.0",

